react-native/react/native/remote-component-loader.js

```diff
@@ -68,35 +68,46 @@ class RemoteComponentLoader extends Component {
     const currentWindow = getCurrentWindow()
 
     currentWindow.on('hasProps', props => {
-      // Maybe we need to wait for the state to arrive at the beginning
-      if (props.waitForState &&
-          // Make sure we only do this if we haven't loaded the state yet
-          !this.state.loaded &&
-          // Only do this if the store hasn't been filled yet
-          Object.keys(this.store.getState()).length === 0) {
-        const unsub = this.store.subscribe(() => {
-          getCurrentWindow().show()
-          getCurrentWindow().setAlwaysOnTop(false)
-          this.setState({props: props, loaded: true})
-          unsub()
-        })
-      } else {
-        // If we've received props, and the loaded state was false, that
-        // means we should show the window
-        if (this.state.loaded === false) {
-          currentWindow.show()
-          currentWindow.setAlwaysOnTop(false)
+      try {
+        // Maybe we need to wait for the state to arrive at the beginning
+        if (props.waitForState &&
+            // Make sure we only do this if we haven't loaded the state yet
+            !this.state.loaded &&
+            // Only do this if the store hasn't been filled yet
+            Object.keys(this.store.getState()).length === 0) {
+          const unsub = this.store.subscribe(() => {
+            getCurrentWindow().show()
+            getCurrentWindow().setAlwaysOnTop(false)
+            this.setState({props: props, loaded: true})
+            unsub()
+          })
+        } else {
+          // If we've received props, and the loaded state was false, that
+          // means we should show the window
+          if (this.state.loaded === false) {
+            currentWindow.show()
+            currentWindow.setAlwaysOnTop(false)
+          }
+          setImmediate(() => this.setState({props: props, loaded: true}))
         }
-        setImmediate(() => this.setState({props: props, loaded: true}))
-      }
+      } catch (_) { }
```

Comment:
This is the subscription to its own store so i don't think its important that the unsub happens (cleanup wise)